tst/CMakeLists.txt

```diff
@@ -1,6 +1,10 @@
 
 add_library(catch2_define catch2_define.cpp)
 target_link_libraries(catch2_define PUBLIC Catch2::Catch2 Kokkos::kokkos)
+if (ENABLE_MPI)
+  target_link_libraries(catch2_define PUBLIC MPI::MPI_CXX)
```

Comment:
do we want to rely on the C++ bindings for MPI? Aren't they completely un-maintained?